core/src/main/java/com/amplifyframework/core/DeveloperMenuActivity.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
2020

Reply:
Should the copyright year in Amplify.java be updated to 2020 as well?

Reply:
The Amazon guideline is to use the date of authorship for the copyright date. So, this one would be 2020, and Amplify would be 2019.